examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -30,15 +30,16 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.                if len(count_i[count_i &lt; core_num]) &lt;= 0 and len(count_u[count_u &lt; core_num]) &lt;= 0:
</pre><p>len cannot be negative, so == 0 is better here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
There is also [PEP8](https://www.python.org/dev/peps/pep-0008/) that recommends removing the `== 0` altogether. I think it would work with a Pandas DF or Series, as here.